src/__swaps__/screens/Swap/hooks/useAssetsToSell.ts

```diff
@@ -1,58 +0,0 @@
-import { useMemo } from 'react';
-
-import {
-  selectUserAssetsList,
-  selectUserAssetsListByChainId,
```

Comment:
can delete `selectUserAssetsListByChainId` function as it is not used anywhere else